packages/kbn-grid-layout/grid/grid_panel.tsx

```diff
@@ -97,18 +72,28 @@ export const GridPanel = ({
             height: ${euiThemeVars.euiSizeL};
             z-index: ${euiThemeVars.euiZLevel3};
             margin-left: ${euiThemeVars.euiSizeS};
-            border: 1px solid ${euiThemeVars.euiBorderColor};
-            background-color: ${euiThemeVars.euiColorEmptyShade};
+            border: 1px solid ${euiTheme.border.color};
+            background-color: ${euiTheme.colors.emptyShade};
```

Comment:
This wasn't responding to dark mode, because it was a static value - fixed this by switching to using the theme hook, so it now responds dynamically:

![Oct-22-2024 09-03-02](https://github.com/user-attachments/assets/3bac6810-7eec-4a4a-99c1-6f23a65f5f95)